### PR TITLE
Translation improvements for language zh_CN

### PR DESCRIPTION
**Pull request by huige233**
**Brief Description of What This PR Does**

This Pr fix some Translated question

**Related Issues**

zh_CN translated about

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by 84634E1A607A** [comment]
Translation changes can be made in [thrive Weblate](https://translate.revolutionarygamesstudio.com/projects/thrive/thrive-game/zh_Hans/).

Please sign thrive CLA, a legal agreement to avoid copyright problems.

File: locale/zh_CN.po [hunk 1]
```diff
@@ -5958,7 +5958,7 @@ msgstr ""
 #: ../src/tutorial/microbe_editor/MicrobeEditorTutorialGUI.tscn:97
 msgid "TUTORIAL_MICROBE_EDITOR_SELECT_ORGANELLE_TEXT"
 msgstr ""
-"在菜单中选择细胞器时，要密切注意弹出的工具提示，了解它们的作用。如果你不考虑它们的功能，零件可能反而会给你带来麻烦。\n"
+"在菜单中选择细胞器时，请务必详读并理解弹出的提示。不考虑其机能的使用可能会使其变得弊大于利。\n"
```
Comment (84634E1A607A) [inline locale/zh_CN.po:5961]: 不如把 "其" 去掉得了. 一堆 "其" 读的头疼.

**Review by 84634E1A607A — COMMENTED** [review]
Overall good. CLA required. One suggestion (made in Chinese).

File: locale/zh_CN.po (hunk 1 above)
Comment (huige233) [reply locale/zh_CN.po:5961]: If you remove the first "其", it will lead to semantic confusion, but it is good to remove the second one

Comment (84634E1A607A) [reply locale/zh_CN.po:5961]: Fine then. Are you in the community discord server? If so, we can discuss about the translations there. My discord name is Ajax.

Comment (huige233) [reply locale/zh_CN.po:5961]: Another point is that I want to use the name "基因组件" for the organelles here

Comment (84634E1A607A) [reply locale/zh_CN.po]: We want to be scientific and accurate. So I'd prefer the original one.

**Review by 84634E1A607A — APPROVED** [review]

**Comment by 84634E1A607A** [comment]
I'd suggest that you use weblate for further translation changes, because it is far more convenient and (mostly) doesn't need to be reviewed. You can also see its English text and be informed of untranslated sentences. (Chinese isn't 100% translated yet for many reasons)

**Pull request merged**